packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -514,6 +531,19 @@ export function TransferPanelMain({
           </BalancesContainer>
         </NetworkListboxPlusBalancesContainer>
       </NetworkContainer>
+
+      <Dialog
```

Comment:
Some final comments: 

1. Let's set some max-width for the dialog, this looks a bit janky:

<img width="1624" alt="Screen Shot 2022-08-11 at 15 46 28" src="https://user-images.githubusercontent.com/20543771/184148135-2f71f7b5-1584-463b-9abf-fe03b366ef2d.png">

We have a couple of classes we used for other dialogs, like `md:max-w-[490px]` or `md:max-w-[628px]`, so you can use those.

2. Let's use the token symbol instead of the token name. `The Arbitrum bridge does not currently support wstETH` sounds cleaner than `The Arbitrum bridge does not currently support Wrapped liquid staked Ether 2.0`.